### ISSUE TITLE
Update the downloads page to link to Logstash plugins instead of Logstash Docker/tar

### ISSUE THREAD

**Issue by dlvenable** [opensearch-project/project-website#2502]
**Is your feature request related to a problem? Please describe.**

The OpenSearch project is no longer maintaining the Logstash Docker image and tar.gz files. The downloads page still has these links. 

**Describe the solution you'd like**

Update the downloads page such that:
* It does not include download options for the Docker image and tar.gz for Logstash
* It has links to the RubyGems for the logstash-output-opensearch and logstash-input-opensearch projects.
* It includes the Docker installation command for users to copy.

**Comment by dblock** [comment]
Catch All Triage - [1](https://github.com/dblock) [2](https://github.com/krisfreedain) [3](https://github.com/gkamat) [4](https://github.com/andrross)

@dlvenable Do we still need this?

**Comment by dlvenable** [comment]
@dblock , Yes, we do still need this to direct users on how to get the latest released version.
